### DISCUSSION TITLE
Tailwind css not appling classes from js file

### DISCUSSION THREAD

**Discussion by dhanashrivitara** [tailwindlabs/tailwindcss#10905] [Help]
my js file having tailwindCss but its is not working

![image](https://user-images.githubusercontent.com/81235677/228601035-3d249fc6-3830-49b9-9e89-c1453a07a1f9.png)

and config file i have added the file path

![image](https://user-images.githubusercontent.com/81235677/228601207-7baa32c3-9502-4479-a7f3-47d55e47a1ae.png)

please help me how classes will work

**Comment by wongjn** [comment]
If you mean "appling classes" meaning that the class names declared in the `class` property of the object argument you are calling `utils.createHTMLElement()` with are not being added to the (what I assume to be) `<select>` element, then this would not be related to Tailwind but the JavaScript that the code executes. Otherwise, if by "appling classes" you mean that the resulting element is not styled as you expect from the class names declared in the `class` property of the object argument you are calling `utils.createHTMLElement()` with, then perhaps double-check the `content` file path is correct.

**Reply by wongjn** [reply]
> Yes that path is correct i have taken from folder that path is accurate but that class i am calling in utils.createHTMLElement() are not added after i am running command for output.css file i have checked but those classes are not in css file.[^1]

Consider providing a remotely-hosted git repository that reproduces the problem. This will allow us to take a closer look and perhaps figure out the problem.

[^1]: https://github.com/tailwindlabs/tailwindcss/discussions/10905#discussioncomment-5468103

**Reply by wongjn** [reply]
> It is a product of company which cannot be share publicly but can you help me with video call or other any solution.[^1]

Consider providing a remotely-hosted git repository that reproduces the problem. It doesn't have to be the one you are working on, but perhaps a similar one that is structured similarly and has similar code.

[^1]: https://github.com/tailwindlabs/tailwindcss/discussions/10905#discussioncomment-5468211

**Reply by dhanashrivitara** [reply]
ok

**Comment by dhanashrivitara** [comment]
Yes that path is correct i have taken from folder that path is accurate but that class i am calling in utils.createHTMLElement() are not added after i am running command for output.css file i have checked but those classes are not in css file.

**Reply by wongjn** [reply]
Replied at https://github.com/tailwindlabs/tailwindcss/discussions/10905#discussioncomment-5468123 to keep the reply chain together.

**Reply by dhanashrivitara** [reply]
It is a product of company which cannot be share publicly but can you help me with video call or other any solution.

**Reply by wongjn** [reply]
Replied at https://github.com/tailwindlabs/tailwindcss/discussions/10905#discussioncomment-5468244 to keep the reply chain together.
